client/packages/invoices/src/OutboundShipment/DetailView/Toolbar.tsx

```diff
@@ -18,12 +18,14 @@ import { useOutbound } from '../api';
 export const Toolbar: FC = () => {
   const onDelete = useOutbound.line.deleteSelected();
   const { onAllocate } = useOutbound.line.allocateSelected();
-  const { otherParty, theirReference, update } = useOutbound.document.fields([
+  const { id, otherParty, theirReference, update } = useOutbound.document.fields([
+    'id',
     'otherParty',
     'theirReference',
   ]);
   const [theirReferenceBuffer, setTheirReferenceBuffer] =
     useBufferState(theirReference);
+  const { mutateAsync: onUpdate } = useOutbound.document.updateName();
```

Comment:
:nit The `onXxxx` naming is usual for events - but this is an explicit action method, and would read better if it said what it does:
```suggestion
  const { mutateAsync: updateName } = useOutbound.document.updateName();
```